src/gui/wizard/owncloudoauthcredspage.h

```diff
@@ -36,25 +39,24 @@ class OwncloudOAuthCredsPage : public AbstractCredentialsWizardPage
     AbstractCredentials *getCredentials() const Q_DECL_OVERRIDE;
 
     void initializePage() Q_DECL_OVERRIDE;
+    void cleanupPage() override;
```

Comment:
AFAIK we dropped centos 6 whi had that old compiler. And even then we had upgraded to a more recent one.